VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -320,6 +327,8 @@ private void checkNoResultView(EmptyQueryResultsFor query) {
                 ((TaskProcessCallback) activity).onMessage(MessageType.ERROR, resultsText);
             } else {
                 ((TaskProcessCallback) activity).onMessage(MessageType.EMPTY, "");
+                spinnersContainerLayout.setVisibility(View.VISIBLE);
+                discussionPostsListView.setVisibility(View.VISIBLE);
```

Comment:
I think these should be done separately from this method, as they're not covered in the method description. Or you could also change the method name and description to be broader I guess.